book/src/language-tour/custom-types.md

```diff
@@ -139,7 +139,7 @@ let Dog { name: name, cuteness: _, age: _ } = dog
 builtin.decode_utf8(name) // "Cashew"
 
 // Other fields ignored by spreading.
-// Field punning is supported. Hence `age` is a shorthand for `age: age`.
+// Field prunning is supported. Hence `age` is a shorthand for `age: age`.
```

Comment:
This is actually meant to be punning. It means when a variable name is the same as a field name you get special sugar.

https://dev.realworldocaml.org/records.html

Reply:
Ok thanks, didn't know that.